content/cloud/serverless-ai.md

```diff
@@ -0,0 +1,72 @@
+- [Quotas And Service Limitations For Fermyon Serverless AI](#quotas-and-service-limitations-for-fermyon-serverless-ai)
+- [FAQ](#faq)
+- [Next Steps](#next-steps)
+
+## Accessing Private Beta
+
+The Fermyon Serverless AI is currently in private beta. To request access to the private beta, please fill out this short [sign-up form](https://fibsu0jcu2g.typeform.com/to/mNzgXRvB#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+ 
+> Please note that the private beta is limited in space, and all requests cannot be guaranteed. 
+
+Once you have access to the private beta, please ensure you have the `canary` version of Spin installed:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash -s -- -v canary
+```
+
+Also, for TypeScript/JavaScript examples, please ensure you have the latest TypeScript/JavaScript SDK installed:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin templates install --git https://github.com/fermyon/spin-js-sdk --upgrade
+```
+
+## Quotas And Service Limitations For Fermyon Serverless AI
+
+*Quotas* 
+* You can have up to 75 inferencing requests per hour, 200 embedding requests per hour
+* You can have a maximum of 1,024 tokens (prompt and response) per request
+
+>> If you have questions about increasing these limits, please reach out to [support@fermyon.com](mailto://support@fermyon.com)
+
+*Service Limitations*
+* Fermyon Serverless AI currently supports the following models: `Code Llama Instruct`, `Llama2 Chat`, and `all-MiniLm-L6-v2`
+
+## FAQ
+
+- **How does Fermyon Serverless AI compare to OpenAI? In what situations am I better off using Fermyon Serverless AI?**
+With Fermyon Serverless AI, you don’t need to be worried about vendor lock-in. With Fermyon Serverless AI you can run your inferencing workloads locally with Spin or on your infrastructure of choice with Spin install. In fact, Fermyon Serverless AI is built using open source models that can be used completely outside the context of Spin in case you decide to host your own infrastructure end to end. 
+
+- **Which specific model(s) is Fermyon Serverless AI using for inferencing? [i.e. 7B, 13B, 70B] Why was this model chosen?**
+Fermyon Serverless AI is using the 13B model for inferencing, both for `llama2-chat` and `codellama-instruct`, as it strikes an excellent balance of performance and accuracy. If you’re interested in another model, please share this feedback at [github.com/fermyon/feedback](github.com/fermyon/feedback).
+
+- **How do I choose which model I’m using?**
+`llama2-chat` is trained for chat use cases, although it can be used for general language model tasks as well. `code llama` is meant for generating code based on instructions. However, this are just high-level guideposts. Please visit [Meta AI’s documentation](https://ai.meta.com/resources/models-and-libraries/llama/) for more specific instructions and to learn more about the tradeoffs between llama2 and Code Llama
```

Comment:
```suggestion
`llama2-chat` is trained for chat use cases, although it can be used for general language model tasks as well. `code llama` is meant for generating code based on instructions. However, these are just high-level guideposts. Please visit [Meta AI’s documentation](https://ai.meta.com/resources/models-and-libraries/llama/) for more specific instructions and to learn more about the tradeoffs between llama2 and Code Llama
```